runtime/src/actor_error.rs

```diff
@@ -61,6 +101,16 @@ impl ActorError {
         &self.msg
     }
 
+    /// Returns the optional data that might be associated with the error
+    pub fn data(&self) -> &[u8] {
+        self.data.as_ref().map_or(&[] as &[u8], |d| d.data.as_slice())
+    }
```

Comment:
I realise this might have been written with some code in mind that we can't see here, but this seems like an oddly specific conversion to inline right here. Discarding the codec on purpose? Why not provide `&Option<IpldBlock>` and let the caller do what they need with it? Or some other variant.

Reply:
As far as I can tell, it's only used in a couple of tests.

(IIRC, it was written this way before we started using `IpldBlock`, and it should definitely be removed).

Reply:
Yup, I've removed it.